src/drivers/mongodb.ts

```diff
@@ -29,7 +29,11 @@ export default defineDriver((opts: MongoDbOptions) => {
       if (!opts.connectionString) {
         throw createRequiredError(DRIVER_NAME, "connectionString");
       }
-      const mongoClient = new MongoClient(opts.connectionString);
+      const mongoOptions: any = {};
+      mongoOptions.driverInfo = {
+        name: "unstorage",
+      };
+      const mongoClient = new MongoClient(opts.connectionString, mongoOptions);
```

Comment:
I think we should instead support generic `mongoOptions` as a driver option so you can set it to `{ driverInfo: { name: "..." } }` based on your needs

Reply:
i agree. we could default if it helps:

```ts
// in the top of the file
const defaultMongoOptions: MongoClientOptions = {
  driverInfo: {
    name: "unstorage"
  }
};

// here (where i'm commenting in this PR)
const mongoOptions: MongoClientOptions = {
  ...defaultMongoOptions, // don't bother deep-merging, people can override `driverInfo` as a whole
  ...opts.clientOptions
};
```

then update the `MongoDbOptions` to have:

```ts
clientOptions?: MongoClientOptions;
```